func_sig_registry/registry/forms.py

```diff
@@ -98,8 +101,19 @@ def create(self, validated_data):
         else:
             num_imported = sum(tuple(zip(*import_results))[1])
             num_duplicates = num_processed - num_imported
+        
+        #Event Signature handler
+        import_results_event = EventSignature.import_from_contract_abi(contract_abi)
+        num_processed_event = len(import_results_event)
+        if num_processed_event == 0:
+            num_imported_event = 0
+            num_duplicates_event = 0
+        else:
+            num_imported_event = sum(tuple(zip(*import_results_event))[1])
+            num_duplicates_event = num_processed_event - num_imported_event
+
         return {
-            'num_processed': num_processed,
-            'num_imported': num_imported,
-            'num_duplicates': num_duplicates,
+            'num_processed': num_processed + num_processed_event,
```

Comment:
I would recommend going ahead and adding a suffix to the function variables as well: ie: `num_processed_functions`